schedules.py

```diff
@@ -0,0 +1,23 @@
+"""Module responsible to handle schedules."""
+import time
+from sched import scheduler
+from kytos.core.helpers import now
+from threading import Thread
+
+
+class Schedule:
+    """Schedule events."""
+
+    def __init__(self):
+        """Initialize the schedule structure."""
+        self.events = scheduler(time.time, time.sleep)
+        self.thread = None
+
+    def add_circuit_event(self, deploy_method, circuit):
+        """Add a new circuit event."""
+        seconds = (circuit.creation_time - now()).total_seconds()
+        self.events.enter(seconds, 1, deploy_method, argument=(circuit,))
+
+        if self.thread is None or self.thread.is_alive() is False:
```

Comment:
In the future we will change this to `asyncio`. @hdiogenes is working on this. Please check with him, if this Thread will break his code.